courses/free-courses-de.md

```diff
@@ -11,9 +13,21 @@
 * [Python](#python)
 * [Rust](#rust)
 * [Spieleentwicklung](#spieleentwicklung)
+* [SQL](#sql)
 * [TypeScript](#typescript)
 
 
+### Ansible
+
+* [Ansible Tutorial](https://www.redhat.com/de/topics/automation/learning-ansible-tutorial) - RedHat
+
+
+### Bash
+
+* [Bash-Scripting Grundkurs](https://www.ernstlx.com/linux90script.html)
+* [LearnShell](https://www.learnshell.org/de/) - Interaktiv die Shell kennenlernen
```

Comment:
couldn't navigate this because too many ads

Reply:
alright, I removed it